designcompose/src/main/java/com/android/designcompose/squoosh/SquooshTreeBuilder.kt

```diff
@@ -566,24 +567,26 @@ private fun generateOverlayNode(
         }
     }
 
-    val nodeStyleBuilder = node.style.nodeStyle.toBuilder()
-    nodeStyleBuilder.overflow = Overflow.OVERFLOW_VISIBLE
-    nodeStyleBuilder.backgroundsList.clear()
-
-    overlay.overlayBackgroundOrNull?.colorOrNull?.let {
-        val bgColor =
-            com.android.designcompose.definition.element.color {
-                r = (it.r * 255.0).toInt()
-                g = (it.g * 255.0).toInt()
-                b = (it.b * 255.0).toInt()
-                a = (it.a * 255.0).toInt()
-            }
-        nodeStyleBuilder.backgroundsList.add(background { solid = colorOrVar { color = bgColor } })
+    val newNodeStyle = node.style.nodeStyle.copy {
+        overflow = Overflow.OVERFLOW_VISIBLE
+        this.backgrounds.clear()
```

Comment:
I would keep the usage of `this` consistent. Either use it everywhere or not at all.